public/parts.gohtml

```diff
@@ -51,6 +51,10 @@
                                    class="btn btn-sm btn-link btn-outline-light bg-dark text-light text-monospace">
                                     click track <i class="far fa-file-audio"></i>
                                 </a>
+                                <a href="{{.BackingTrack}}"
```

Comment:
Is this correct?

Reply:
Yes, this does render correctly. I'm currently working on infra to show off these pr's

Reply:
Yeah, I misread the context on first read.